packages/terminal-extension/src/index.ts

```diff
@@ -252,61 +261,107 @@ export function addCommands(
   commands.addCommand(CommandIDs.refresh, {
     label: 'Refresh Terminal',
     caption: 'Refresh the current terminal session',
-    execute: () => {
+    execute: async () => {
       let current = tracker.currentWidget;
       if (!current) {
         return;
       }
       app.shell.activateById(current.id);
-
-      return current.content.refresh().then(() => {
+      try {
+        await current.content.refresh();
         if (current) {
           current.content.activate();
         }
-      });
+      } catch (err) {
+        Private.showErrorMessage(err);
+      }
     },
     isEnabled: () => tracker.currentWidget !== null
   });
 
-  function showErrorMessage(error: Error): void {
-    console.error(`Failed to set ${plugin.id}: ${error.message}`);
-  }
-
   commands.addCommand(CommandIDs.increaseFont, {
     label: 'Increase Terminal Font Size',
-    execute: () => {
-      let { fontSize } = Terminal.defaultOptions;
+    execute: async () => {
+      let { fontSize } = ITerminal.defaultOptions;
       if (fontSize < 72) {
-        return settingRegistry
-          .set(plugin.id, 'fontSize', fontSize + 1)
-          .catch(showErrorMessage);
+        try {
+          await settingRegistry.set(plugin.id, 'fontSize', fontSize + 1);
+        } catch (err) {
+          Private.showErrorMessage(err);
```

Comment:
This will need to be consumed by the `void` operator with the new tslint settings. Sorry!

Reply:
I _think_ i fixed this correctly...

Reply:
is this right now?

Reply:
Oh, shoot, my mistake. I thought this was using the apputils `showErrorMessage`, which returns a promise. Since this one is synchronous, no special handling is required.